pages/addContact.vue

```diff
@@ -104,7 +110,16 @@
    const emailAddress = ref('');
```

Comment:
instead of a ref for every possible property, you can do it all in one object:

`
const data = ref({
prefix: '',
firstName: ''
})`


and in the template: `v-model="data.prefix"`

and then when sending data to the backend its just 

`{
body: data.value
}`